tests/main/install-closed-channel/task.yaml

```diff
@@ -0,0 +1,9 @@
+summary: Check that installing a snap from a closed channel DTRT
+
+details: |
+    When a snap is installed from a closed channel, we're supposed to
+    "forward" to the next available risk level.
+
+execute: |
+    snap install --beta test-snapd-tools | MATCH '^Channel beta for test-snapd-tools is closed; temporarily forwarding to stable\.$'
+    snap info test-snapd-tools | MATCH tracking:.*beta
```

Comment:
Don't think it's strictly required, but the extra quotes would be nice: `'tracking:.*beta'` may

Can you list ` tests/main/install-closed-channel/task.yaml` in `tests/unit/spread-shellcheck/must`?